### DISCUSSION TITLE
Is there a upgrade planned to replace grpc dependency with @grpc/grpc-js ?

### DISCUSSION THREAD

**Discussion by Prashvith** [stephenh/ts-proto#359] [Q&A]
When generating typescript file with metadata there is an import of metadata from 'grpc' plugin added to the files. This mean 'grpc' dependency needs to be present in the project. How to solve this if I want to use @grpc/grpc-js in my project

**Comment by stephenh** [comment]
Hi! I don't personally have the time to fix this right now, but there is an open issue about this, #356; a PR would be great. Thanks!
